src/pages/SignUp/SignupForm/UserTypeSelector.tsx

```diff
@@ -0,0 +1,43 @@
+import { useController, useFormContext } from "react-hook-form";
+import { FormValues } from "../types";
+
+const btnClass = (isSelected: boolean) =>
+  `flex items-center justify-center btn-outline-2 h-[42px] ${
+    isSelected
+      ? "bg-[#EAF3FA] border-[1.25px] enabled:border-blue-d1 enabled:hover:border-blue-d1 hover:cursor-default enabled:text-blue-d1 font-bold"
```

Comment:
@SovereignAndrey, existing `blue-l4` (from standard provided) could also be used in place of this stray `#EAF3FA` as they really just look the same

<img width="676" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/5f249bb4-2768-44dd-8889-881229e0c578">